### DISCUSSION TITLE
IE11 issue

### DISCUSSION THREAD

**Discussion by joshyboyrules** [react-hook-form/react-hook-form#1962] [Q&A] [answered]
Hello! 

We are running into a IE11 issue using `react-hook-form@6.0.0-rc.5` and
```
import {useForm} from 'react-hook-form/dist/react-hook-form.ie11.js';
```

Getting these errors. 
![Screen Shot 2020-06-25 at 9 37 30 AM](https://user-images.githubusercontent.com/38472898/85730392-9e5f1500-b6c7-11ea-998a-04ef1f5aaaa3.png)
![Screen Shot 2020-06-25 at 9 36 53 AM](https://user-images.githubusercontent.com/38472898/85730396-9f904200-b6c7-11ea-8bcf-7d1811436520.png)

Just curious as to how y'all would try to tackle this. We are currently on `corejs: 2.x.x` so the error makes sense but we don't see `corejs:3` listed as a peer dependency for react-hook-form. Any help would be much appreciated. 

As always thanks for this awesome form library 😄 .

**Comment by bluebill1049** [comment, marked as answer]
we are using core 3 for a long time actually 
<img width="558" alt="Screen Shot 2020-06-26 at 9 02 28 am" src="https://user-images.githubusercontent.com/10513364/85803985-cd3ebf00-b78b-11ea-9d2f-e63cdcd96755.png">
